app/models/user.js

```diff
@@ -25,3 +29,16 @@ module.exports = (sequelize, DataTypes) =>
     },
     { underscored: true }
   );
+
+  User.createUser = user =>
+    User.create(user).catch(error => {
+      const { message, errorFn } = dbErrorCodes[error.name] || { message: 'Unable to create new user.' };
```

Comment:
Be careful with this mapping, sometimes is useful access directly to error object, before mapping log error object for get more detail of error